.github/workflows/my-ci-pipeline.yml

```diff
@@ -19,4 +17,34 @@ jobs:
     - run: ls
     - run: touch .env # creating empty env file coz docker compose needs one
     - run: docker build --target test -t todo-app:test . 
-    - run: docker run --env-file ./.env.test todo-app:test
\ No newline at end of file
+    - run: docker run --env-file ./.env.test todo-app:test
+
+  dockerhub:
+    name: Build and image
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.ref_name == 'module8_updated'
```

Comment:
Potentially something you've already had to worry about with m9, but don't forget to update this (generally to `main`, or to specific branches if testing)